frontend/src/features/public-form/components/FormFields/PublicFormSubmitButton.tsx

```diff
@@ -42,7 +42,7 @@ export const PublicFormSubmitButton = ({
   }, [formInputs, formFields, formLogics])
 
   return (
-    <Stack px={{ base: '1rem', md: 0 }} pt="2.5rem" pb="4rem">
+    <Stack px={{ base: '1rem', md: 0 }} pt="2.5rem" pb="1.5rem">
```

Comment:
should we use a margin here instead so we can use margin collapse on the switcher so we do not need to remember to change this when we remove the switcher?

```tsx
<Stack px={{ base: '1rem', md: 0 }} mt="2.5rem" mb="4rem">
...
</Stack>

// Then in the switch env message component
// -2.5rem so the total spacing between the components is still 1.5rem
<Box mt="-2.5rem">...</Box>
```

Reply:
[9c49ab2](https://github.com/opengovsg/FormSG/pull/5561/commits/9c49ab2fe57d9b14eb9ec7ae9d03c69116e28023)